Source/UIView+SimpleAnimation.swift

```diff
@@ -71,6 +71,29 @@ public extension UIView {
       }, completion: completion)
     return self
   }
+  
+/**
+     Fades the background color of a view from one color to another without using alpha values. 
+     
+     - Parameters:
+     - fromColor: the starting color you want to begin fade with
+     - toColor: the final color you want to fade to
+     - duration: duration of the animation, in seconds
+     - delay: delay before the animation starts, in seconds
+     - completion: block executed when the animation ends
+     */
+    @discardableResult func fadeColor(fromColor: UIColor = UIColor.white,
```

Comment:
Or, maybe it would be more straightforward just to make fromColor nullable and default it to nil.